docs/design/introduction/styles/iconography.md

```diff
@@ -6,19 +6,19 @@ Tizen uses simple icon designs with subtle shadows. The following are guidelines
 
  
```

Comment:
There is a special character here. 0xC2. 

There is a special character where 3 blank lines are in this file. 
Please remove all special characters.